mitosheet/src/components/taskpanes/FillNa/FillNaTaskpane.tsx

```diff
@@ -0,0 +1,321 @@
+            }
+        })
+    }
+
+    return (
+        <DefaultTaskpane>
+            <DefaultTaskpaneHeader
+                header='Fill NaN Values'
+                setUIState={props.setUIState}   
+            />
+            <DefaultTaskpaneBody>
+                <Row justify='space-between' align='center' title='Select the dataframe to fill nan values in.'>
+                    <Col>
+                        <p className='text-header-3'>
+                            Dataframe
+                        </p>
+                    </Col>
+                    <Col>
+                        <Select
+                            value={props.sheetDataArray[params.sheet_index]?.dfName}
+                            onChange={(newDfName: string) => {
+                                const newSheetIndex = props.sheetDataArray.findIndex((sheetData) => {
+                                    return sheetData.dfName == newDfName;
+                                })
+                                
+                                if (newSheetIndex >= 0) {
+                                    setParams(prevParams => {
+                                        return {
+                                            ...prevParams,
+                                            sheet_index: newSheetIndex
+                                        }
+                                    })
+                                }
+
+                            }}
+                            width='small'
+                        >
+                            {props.sheetDataArray.map(sheetData => {
+                                return (
+                                    <DropdownItem
+                                        key={sheetData.dfName}
+                                        title={sheetData.dfName}
+                                    />
+                                )
+                            })}
+                        </Select>
+                    </Col>
+                </Row>
+                <Spacer px={15}/>
+                <Row justify='space-between' align='center' title='Select the dataframe to fill nan values in.'>
+                    <Col>
+                        <p className='text-header-3'>
+                            Columns to Fill NaN Values In
+                        </p>
+                    </Col>
+                </Row>
+                <MultiToggleBox
+                    searchable
+                    toggleAllIndexes={toggleIndexes}
+                    height='medium'
+                >
+                    {Object.entries(columnDtypeMap).map(([columnID, columnDtype], index) => {
+                        const columnHeader = columnIDsMap[columnID];
+                        const toggle = params.column_ids.includes(columnID);
+                        const disabled = (params.fill_method.type === 'mean' || params.fill_method.type === 'median') && 
+                            (!isNumberDtype(columnDtype));
```

Comment:
I love adding this check here instead of adding an error message. But we also need this check in the toggleAll. If I:
1. Set the fill_method to median
2. Press toggle all, it will select non-number values
3. Press submit, I will get an error. 

The error doesn't open the error modal, but it prints to the python console. it also still fills in the nan values in the number columns